### PR TITLE
Gives Vasilissans a language finally / Bug-Speak

### PR DESCRIPTION
**Pull request by HungryMoth**
Adds Vespinae: A common language designed for creatures without traditional mouths and tongues. Mandibles, pincers, clicky-clackery-doos.

They have been developed primarily as an extension upon Vasilissan's unique dialect, and as a result has become a psuedo-common tongue among creatures with non-traditional mouthparts!

So with the recent addition of abdomens, some antennae, and my general love for the insectoid life. I feel that they could use a language.

Not whitelisted, normal language, mostly for fluff and lore. It gives Vasilissans, a species that has never had a language while everyone else has sone. They get one finnaly! Yay! And for other custom species they have a 'trope' language they can fall back on to use, and it generally helps with character development, well we don't seem to have anything for our spooky insect friends, so here it is.

A new common language that the Vassilians may use.

Works just fine, tested and compiled on my end, and here is what it looks like, very click-clacky.
https://gyazo.com/1d522224cd1371860ff556b429b3577a

First pull request here so, uh, yeah!

### CODE REVIEW

File: code/modules/mob/language/station_vr.dm [hunk 1]
```diff
@@ -70,6 +70,18 @@
 	syllables = list("salve","sum","loqui","operatur","iusta","et","permittit","facere","effercio","pluribus","enim","hoc",
 	"mihi","wan","six","salve","tartu")
 	machine_understands = FALSE
+	
+/datum/language/bug
+    name = LANGUAGE_VESPINAE
```
Comment (KasparoVy) [inline code/modules/mob/language/station_vr.dm]: # WE REQUIRE MORE VESPENE GAS

Comment (HungryMoth) [reply code/modules/mob/language/station_vr.dm]: Fun Fact: That name comes from "Vespid" which is the taxonomic order of wasps! Not that the language was only for wasps or anything, but _The more you know!_

**Comment by SpadesNeil** [comment]
I would rather not just slap in a generic insect species without an insect race. That said, we already have wasp bits. We could totally add in such a bug race.

Even though bugs bother the hell out of me. :S But we'd be setting a bad precedent if we just added languages with no lore or race to feature them.

**Comment by Screemonster** [comment]
Well, we _do_ have vasillisans or however the fuck they're spelled.

**Comment by HungryMoth** [comment]
We do have Vasillians and they don't have a language at all, I can retool the wording and lore so it is a Language for them.

That would solve the problem of a lore race that doesn't have a language getting one, and still fill in the goal of giving custom species an option.

**Comment by SpadesNeil** [comment]
Let's do that, then.

**Comment by HungryMoth** [comment]
I made the necessary changes and changed the title as well.

What this does now is give Vasilissans their own language by default, and updates the lore/flavor-text that describes it as being developed by them.

Probably was needed for a while as I believe Vasilissans are the only lore-species we have (Besides Xenochimera) that don't have their own language.

I hope this is acceptable now.

**Comment by Heroman3003** [comment]
I don't think any species has secondary language added outside their own, so since Vespinae is becoming their official one, I think removing Sol Common from them would make sense, no?

**Comment by HungryMoth** [comment]
You are right and I considered doing the same thing, but I for one don't know why Sol Common was there in the first place. I opted instead to just leave Sol Common behind unless whoever is in charge of the lore wants me to move that.

**Comment by Heroman3003** [comment]
I think it may have been a placeholder, considering they are the only race around without defined language and just were given a 'something'.

**Comment by HungryMoth** [comment]
I suppose you're right. I removed it, no reason why extra-terrestrial space spiders would understand Sol Common naturally.

**Pull request merged**